### PR TITLE
Add donut logo to homepage

### PR DESCRIPTION
**Pull request by ruben-aguilar**
Adding donut's logo to the homepage: 

Screenshot 2022-12-21 204711

### CODE REVIEW

File: layouts/index.html
Comment (gonzaloserrano) [inline layouts/index.html:34]: What do you think about making it link to the donut homepage?

**Review by gonzaloserrano — APPROVED** [review]

File: layouts/index.html
Comment (ruben-aguilar) [reply layouts/index.html]: Good idea! Updated!

**Comment by ruben-aguilar** [comment]
By the way, I cannot merge the PR since I don't have write access to the repo 😢

File: layouts/index.html
Comment (gonzaloserrano) [reply layouts/index.html]: Sorry to nitpick, but it would be great to center it in mobile:

image

Comment (gonzaloserrano) [reply layouts/index.html]: You know what: forget it. I'll add a h2 or h3 header tomorrow above it.

**Pull request merged**
